indexd/index/blueprint.py

```diff
@@ -220,9 +220,19 @@ def get_index_record(record):
     Returns a record.
     """
 
-    ret = blueprint.index_driver.get(record)
+    # check redis
+    json_record = flask.current_app.redis_client.get(record)
+    if json_record:
+        json_record = json.loads(json_record)
+    else:
+        # get from db
```

Comment:
in fence i do: `check redis; if not in redis: call indexd` so i think we could skip checking redis again here. 3 cases:
- request to indexd from another service: we can assume the other service already checked the cache
- request from a user: now that the request reached indexd, it's too late to improve performance
  - the record is in redis: getting from redis instead of the DB doesn't improve performance (i think)
  - it's not in redis: performance is decreased since indexd checks both